test/jb_projects/challenge_handle.test.js

```diff
@@ -0,0 +1,164 @@
+import { expect } from 'chai';
+import { ethers } from 'hardhat';
+import { getTimestamp } from '../helpers/utils';
+
+import { deployMockContract } from '@ethereum-waffle/mock-contract';
+
+import jbOperatoreStore from '../../artifacts/contracts/JBOperatorStore.sol/JBOperatorStore.json';
+
+describe('JBProjects::challengeHandle(...)', function () {
+  const PROJECT_HANDLE = 'PROJECT_1';
+  const PROJECT_HANDLE_NOT_TAKEN = 'PROJECT_2';
+  const PROJECT_HANDLE_NOT_TAKEN_2 = 'PROJECT_3';
+  const METADATA_CID = '';
+  const PROJECT_ID = 1;
+
+  async function setup() {
+    let [deployer, projectOwner, ...addrs] = await ethers.getSigners();
+
+    let mockJbOperatorStore = await deployMockContract(deployer, jbOperatoreStore.abi);
```

Comment:
I see no tests here that test permissions? This is crucial for every API, please add them.